msteams-platform/graph-api/proactive-bots-&-messages/graph-proactive-bots-&-messages.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Use Microsoft Graph to enable proactive bot installation and messaging in Teams
+description: Describes proactive messaging in Teams and how to implement.
+localization_priority:  Normal
+author: laujan
+ms.author: lajanuar
+ms.topic: Overview
+keywords: teams proactive messaging chat installation Graph
+---
+# Enable proactive bot installation and messaging in Teams with Microsoft Graph(Private Preview)
+
+>[!NOTE]
+> Microsoft Graph private preview permissions may not achieve general availability status and currently should not be used in production.
+
+## What is proactive messaging in Teams?
+
+A proactive messages are initiated by bots to start conversations with a users. They serve many purposes including sending welcome messages, conducting surveys or polls, and sending organization-wide notifications.  Proactive messages in Teams can be delivered as either **ad-hoc** or **dialog-based** conversations:
```

Comment:
The ad hoc vs dialog-based feels out of context in the intro. I've never had to think about this distinction when building bots, at least not in those terms, and I'm struggling to think of a practical example where you mix proactive messages with dialog-based flows.

Reply:
Those terms are mentioned in most of the MS proactive messaging articles so I thought they would be relevant in this context.